modules/gitops-release-notes-1-4-3.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-4-3_{context}"]
+= Release notes for {gitops-title} 1.4.3
+
+[role="_abstract"]
+{gitops-title} 1.4.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="fixed-issues-1-4-3_{context}"]
+== Fixed issues
+
+The following issue has been resolved in the current release:
+
+* Before this update, the TLS certificate in the `argocd-tls-certs-cm` configuration map was deleted by the {gitops-title} unless the certificate was configured in the ArgoCD CR specification `tls.initialCerts` field.link:https://issues.redhat.com/browse/GITOPS-1725[GITOPS-1725]
```

Comment:
**Before this update, the TLS certificate in the argocd-tls-certs-cm configuration map used to get reset by any changes made in ArgoCD CR specification tls.initialCerts field. With this update, you can add the needed certificates into ArgoCD CR specification tls.initialCerts and it will be appended in argocd-tls-certs-cm configuration map**

@iam-veeramalla please suggest

Reply:
Hey @varshab1210 , @DebarghoGhosh is right.

The below line in your comment is not true.
> With this update, you can add the needed certificates into ArgoCD CR specification tls.initialCerts and it will be appended in argocd-tls-certs-cm configuration map